File: app/java/src/main/java/io/infralovers/nomadvaultmysql/model/CustomerInput.java
package io.infralovers.nomadvaultmysql.model;

public class CustomerInput {
    private Integer custNo;
    private String  birthDate  = "";
    private String  firstName  = "";
    private String  lastName   = "";
    private String  createDate = "";
    private String  ssn        = "";
    private String  ccn        = "";
    private String  address    = "";
    private String  salary     = "";

    public Integer getCustNo()   { return custNo; }
    public void    setCustNo(Integer custNo) { this.custNo = custNo; }

    public String  getBirthDate()  { return birthDate; }
    public void    setBirthDate(String birthDate) { this.birthDate = birthDate; }

    public String  getFirstName()  { return firstName; }
    public void    setFirstName(String firstName) { this.firstName = firstName; }

    public String  getLastName()   { return lastName; }
    public void    setLastName(String lastName) { this.lastName = lastName; }

    public String  getCreateDate() { return createDate; }
    public void    setCreateDate(String createDate) { this.createDate = createDate; }

    public String  getSsn()        { return ssn; }
    public void    setSsn(String ssn) { this.ssn = ssn; }

    public String  getCcn()        { return ccn; }
    public void    setCcn(String ccn) { this.ccn = ccn; }

    public String  getAddress()    { return address; }
    public void    setAddress(String address) { this.address = address; }

    public String  getSalary()     { return salary; }
    public void    setSalary(String salary) { this.salary = salary; }
}

File: app/dotnet/Models/CustomerInput.cs
using Microsoft.AspNetCore.Mvc;

namespace NomadVaultMySqlDotnet.Models;

public sealed class CustomerInput
{
    [FromForm(Name = "cust_no")]
    public int? CustNo { get; set; }

    [FromForm(Name = "birth_date")]
    public string BirthDate { get; set; } = string.Empty;

    [FromForm(Name = "first_name")]
    public string FirstName { get; set; } = string.Empty;

    [FromForm(Name = "last_name")]
    public string LastName { get; set; } = string.Empty;

    [FromForm(Name = "create_date")]
    public string CreateDate { get; set; } = string.Empty;

    [FromForm(Name = "ssn")]
    public string Ssn { get; set; } = string.Empty;

    [FromForm(Name = "ccn")]
    public string Ccn { get; set; } = string.Empty;

    [FromForm(Name = "address")]
    public string Address { get; set; } = string.Empty;

    [FromForm(Name = "salary")]
    public string Salary { get; set; } = string.Empty;
}
